library/Zend/Session.php

```diff
@@ -321,6 +321,12 @@ public static function regenerateId()
         } else {
             if (!self::$_unitTestEnabled) {
                 session_regenerate_id(true);
+
+                // This can be removed once we update to 7.4 or above as then
+                // we can use php.ini directive to achieve the same `session.cookie_samesite=Lax`
```

Comment:
I think this comment should specify session.cookie_samesite=None.

Reply:
Fixed.